_posts/2024-08-02-gsoc2024_talagayev.md

```diff
@@ -0,0 +1,193 @@
+Linux, MacOS and Windows systems with the Python versions 3.10, 3.11 and 3.12 with different `IpyWidgets` versions, with the oldest supported version for this code being `7.6.4`. Finally,
+a documentation based on the MDAKits Template was created and can be found here: https://mdonatello.readthedocs.io/
+
+## Current State
+
+The current state of MDonatello is, that it is available and working, thus it can be easily installed and used on your system. Some improvements can be made, but overall
+the current version is stable and applicable.
+
+## What's left to do
+
+The code is working, but there is always things that can be improved. The main parts of the code that would need to be improved would be the functional groups, since currently the SMART
+Patterns are not optimal and it leads to cases of functional groups overlapping, an example being the recognition of an hydroxyl group in an carboxylic acid group, thus such cases need to
+be separated. Additionally, the figure generation is not optimal, since currently it uses `Draw.MolToImage`, which during the discussion with the mentors was decided as an not so good option
+and it would need to be adjusted. Additionally, the **Partial Charge Heatmap** is currently mainly highlighting the atoms, but there is an option to display it as a heatmap, which would require
+ code adjustment. Then the `update_display` is also not optimal currently and would need to be improved. There are also some small code structure details, that would need to be adjusted for
+the code to be more clean and structured and last, but not least the code would need to be moved into the `MDAKits` and also published in `conda-forge` and `pypi` to make it more accessible.
+
+## What code got merged (or not) upstream
+
+There were multiple PRs created for this project, but they were merged to obtain a big and summarized [PR](https://github.com/talagayev/MDonatello/pull/1), which is also the version that is 
+used in the [Release](https://github.com/talagayev/MDonatello/releases/tag/0.0.1)
+
+## How to use it
+
+Currently `MDonatello` is a separate package that can be git cloned and installed with the following steps:
+
+First clone the repository:
+
+```
+git clone https://github.com/talagayev/MDonatello
+```
+
+Create a virtual environment and activate it:
+
+```
+conda create --name mdonatello
+conda activate mdonatello
+```
+
+Then go into the MDonatello folder:
+
+```
+cd MDonatello
+```
+
+Finally this package from source:
+
+```
+pip install -e .
+```
+
+To use the **mdonatello** package you need to run a jupyter notebook, thus run the command:
+
+
+```
+jupyter notebook
+```
+
+Now that you started a jupyter notebook create a notebook file and enter the following command to use **mdonatello**:
+Here you need to adjust the name of the PDB File to your PDB File and the resname of the molecule to your molecule
+
+```python
+import MDAnalysis as mda
+import mdonatello
```

Comment:
That import statement shouldn't be needed since you're importing the class directly in the line below?

Reply:
yes you are right, somehow I usually import the package in a separate line and then the class that I use (more of a habit that I have in coding), but I can remove it :)